runtime/CSharp/runtime/CSharp/Antlr4.Runtime/Properties/AssemblyInfo.cs

```diff
@@ -1,49 +0,0 @@
-﻿/* Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
- * Use of this file is governed by the BSD 3-clause license that
- * can be found in the LICENSE.txt file in the project root.
- */
-using System;
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("Antlr4.Runtime")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Antlr organization")]
-[assembly: AssemblyProduct("Antlr4.Runtime")]
-[assembly: AssemblyCopyright("Antlr organization")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-[assembly: CLSCompliant(true)]
-
-#if !PORTABLE || NET45PLUS
-// Setting ComVisible to false makes the types in this assembly not visible
-// to COM components.  If you need to access a type in this assembly from
-// COM, set the ComVisible attribute to true on that type.
-[assembly: ComVisible(false)]
-
-#if !PORTABLE
-// The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid("bc228eb9-e79c-4e5a-a1b9-0434ea566bab")]
-#endif
-#endif
-
-// Version information for an assembly consists of the following four values:
-//
-//      Major Version
-//      Minor Version
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("4.9")]
```

Comment:
I'm not sure if all these things are still used, but I noticed that `AssemblyVersion` changed from 4.9.0.0 in 4.9.0 to 0.0.0.0 in 4.9.1
Here are screenshots from [Package Explorer]
(https://github.com/NuGetPackageExplorer/NuGetPackageExplorer)

4.9.0
![image](https://user-images.githubusercontent.com/919634/105714677-efb0da80-5f1c-11eb-81a7-b032df813671.png)

4.9.1
![image](https://user-images.githubusercontent.com/919634/105714627-e0319180-5f1c-11eb-8dbf-99bcb14657b2.png)

Reply:
@KvanTTT and @ericvergnaud. oh no!  did I miss updating version number?  Can you tell me if this is catastrophic or whether we can leave things as they are until the next release?